cdap-ui/app/features/hydratorplusplus/controllers/create/toppanel-ctrl.js

```diff
@@ -64,8 +45,27 @@ class HydratorPlusPlusTopPanelCtrl{
       this.openMetadata();
     }
 
+    this.isPreviewEnabled = MY_CONFIG.hydrator.previewEnabled === 'true' || MY_CONFIG.hydrator.previewEnabled === true;
```

Comment:
I am also not sure why we are doing both. Shouldn't be either only `true` or `"true"`? Am I missing something here?